share/dbtypes_gen.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+
+with open(os.path.join(os.path.dirname(__file__), '..', 'src', 'query_backend.h')) as f:
+    lines = f.readlines()
+
+VERSION = "v"+sys.argv[1]
+
+ALL_BACK = ["SQLite", "HDF5"]
+
+ENUM_START = 0
+
+for i in range(0,len(lines)):
+    if "enum DbTypes {" in lines[i]:
+        ENUM_START = i
+
+ID_INDEX = 0
+DB_INDEX = 1
+CPP_INDEX = 2
+SHAPE_INDEX = 3
+BACK_INDEX = 4
+VERSION_INDEX = 5
+SUPPORTED_INDEX = 6
+CANON_INDEX = 7
+VL_INDEX = 8
+
+def main():
+    outer = []
+    for back in ALL_BACK:
+        ID = 0
```

Comment:
not a constant variable, should be lowercase.  However, `id` is a built-in Python function, so you may want to use another name.